docs/topics_5/exclude-files-and-packages.adoc

```diff
@@ -1,10 +1,10 @@
 // Module included in the following assemblies:
 // * docs/cli-guide_5/master.adoc
 [id='exclude_files_and_packages_{context}']
-= Configuring {ProductShortName} to Exclude Files and Packages
+= Configuring {ProductShortName} to exclude files and packages
 
```

Comment:
Just a comment about word order. If "Excluding packages" precedes "Excluding files", might be better to reverse the order in this title.

Reply:
Done.